docs/integrations/notification/index.rst

```diff
@@ -0,0 +1,126 @@
+#####################################
+ Monitor Experiment Through Webhooks
+#####################################
+
+This section includes user guides for how to set up notifications for experiments.
+
+Determined now supports webhooks for sending notification to monitor experiment state change.
+
+**********
+ Security
+**********
+
+Determined utilizes ``signed_payload`` and ``timestamp`` headers. You will need to configure a
+``webhook_signing_key`` in master configuration.
+
+.. code::
+
+   security:
```

Comment:
I added a comment on the ticket about more info you can provide here regarding security.